### PR TITLE
Feature/setting backend base url

### PR DESCRIPTION
**Pull request by Iyadbitar**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1 +1 @@
-export const backedAddress = process.env.BACKEND_ADDRESS || '/api'
+export const backedAddress = process.env.REACT_APP_BACKEND_URL || '/api'
```
Comment (jgustie) [inline src/config.js]: Can we make this default to `/v1` so it doesn't need to be specified? We won't be using `/api` going forward.

Comment (Iyadbitar) [reply src/config.js]: I updated this to be /v1 as default but kept it configurable through env var

Comment (Ofer-Idan) [reply src/config.js]: @jgustie you ok with this?

Comment (jgustie) [reply src/config.js]: We can set it for now, but it should change over going forward.

Comment (Ofer-Idan) [reply src/config.js]: Your call. If it's better to not have it configurable and set it to `v1`, let's do that.

Comment (jgustie) [reply src/config.js]: It looks like the default is updated, we are commenting on outdated changes...

**Pull request merged**